test/skimNtuple_HHbtag.cpp

```diff
@@ -226,8 +226,8 @@ int main (int argc, char** argv)
   }
   
   // for boosted corrections
-  bool isDYlike = argv[34];
-  bool isTTlike = argv[35];
+  int isDYlike = atoi(argv[34]);
+  int isTTlike = atoi(argv[35]);
```

Comment:
The name ```isDY/TTlike``` clearly refers to a yes/no answer, i.e. a boolean. ```if (isDYlike)``` is also cleaner than ```if  (isDYlike==1)```. Why is the change needed or desirable?

Reply:
In the previous version, it didn’t behave as expected. This way, it’s more obvious what values it takes, it works, and I think it's ok like this

Reply:
For curiosity, what do you mean by "it didn't behave as expected"? But if this works, it's fine for me.